ev3dev2/unit.py

```diff
@@ -0,0 +1,213 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015 Ralph Hempel <rhempel@hempeldesigngroup.com>
+# Copyright (c) 2015 Anton Vanhoucke <antonvh@gmail.com>
+# Copyright (c) 2015 Denis Demidov <dennis.demidov@gmail.com>
```

Comment:
Ralph is listed in the LICENSE file so I will leave him to be consistent with LICENSE and chop the others.  I do not know what we should legally be doing here...I am not listed in the copyright in any file other than ``debian/copyright`` I suppose that one rules over all the others?

Reply:
I guess since this package is distributed as source code, who is listed in this file doesn't matter too much since the source code files will have the copyright statements as well, which keeps us in compliance with the license terms.

If you want to claim copyright on any file where you made substantial changes, you are certainly free to do so.